scripts/qa_crowbarsetup.sh

```diff
@@ -2150,6 +2150,9 @@ function enable_ssl_generic
         keystone)
             $p "$a['ssl']['ca_certs']" "'/etc/ssl/ca-bundle.pem'"
             $p "$a['api']['protocol']" "'https'"
+            if iscloudver 7minus ; then
+                $p "$a['signing']['ca_certs']" "'/etc/ssl/ca-bundle.pem'"
```

Comment:
is this really in addition to `"$a['ssl']['ca_certs']"` ?

Reply:
Yes. This is confusing :o) The `ssl` section is for https while the `signing` section is for PKI token signing. They default to the same files but only the one under `ssl` is changed by `enable_ssl_generic`.